src/helpers.ts

```diff
@@ -198,3 +198,12 @@ interface SurvivalGameInfo {
      */
     wave: number;
 }
+
+interface _Constructor<T> {
+    readonly prototype: T;
```

Comment:
We'll go through with it then until Typescript can better represent such prototypes. Thank you for your analysis.